test/visual/run-visual-tests.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-require('loud-rejection');
+require('loud-rejection'); // eslint-disable-line import/no-unassigned-import
```

Comment:
Yes, in case of rejected promises, since we don't manually catch them all. This is easier.